<launch>

<include file="$(find camera)/launch/camera.launch"/>

  <group ns="jetbot">
    <node pkg="color_mask" name="color_mask_node" type="color_mask.py"/>
  </group>

</launch>
